hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoCreateSuite.java

```diff
@@ -104,66 +104,60 @@ public class CryptoCreateSuite {
 
     @HapiTest
     final Stream<DynamicTest> idVariantsTreatedAsExpected() {
-        return defaultHapiSpec("idVariantsTreatedAsExpected")
-                .given()
-                .when()
-                .then(submitModified(withSuccessivelyVariedBodyIds(), () -> cryptoCreate("account")
-                        .stakedAccountId("0.0.3")));
+        return hapiTest(submitModified(
```

Comment:
Every `@HapiTest` should now have an `@DisplayName` as per the **HapiTest Checklist** here: https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/test-clients/README.md#the-hapitest-checklist

Reply:
Also where possible try to link the javadoc in the test to the section in the [Hip-796](https://github.com/hashgraph/hedera-improvement-proposal/blob/main/HIP/hip-796.md) outlining the changes in behavior from the HIP